source/slang/slang-ir.cpp

```diff
@@ -6019,3 +6019,15 @@ namespace Slang
     }
 } // namespace Slang
 
+#if SLANG_VC
+#ifdef _DEBUG
+// Natvis sometimes cannot find enum values.
+// Export symbols for them to make sure natvis works correctly when debugging external projects.
```

Comment:
Why not put these symbols into a namespace (like say SlangDebug )? Then they could be (say) `SlangDebug::kIROpNameHint`. Or if you don't want to use a namespace as you are worried about implication of name export/mangling how about `Slang_kIROpNameHint.

That the __ usage is worrying as

https://en.cppreference.com/w/cpp/language/identifiers


```
     the identifiers with a double underscore anywhere are reserved;
    the identifiers that begin with an underscore followed by an uppercase letter are reserved;
    the identifiers that begin with an underscore are reserved in the global namespace. 
```